public/app/frontend/base.scss

```diff
@@ -40,6 +40,13 @@ a, .link {
     @include link;
 }
 
+ul {
+    list-style: none;
+    margin:0;
+    padding: 0;
+    text-indent: 0;
+}
+
```

Comment:
If we set this globally - we might end up having to override the reset when we get to styling 'Rich Text' page content.

I'm don't have strong views either way, but something to consider, if you haven't already :)

Reply:
@EarthlingDavey that's a fair point. I tend to prefer resetting ol as it's so often used in nav structures and in the rich text PR I've set up the correct styling there.